src/accountsdb/snapshots.zig

```diff
@@ -829,64 +829,64 @@ pub const BankFields = struct {
+        const hard_forks = try HardForks.initRandom(random, allocator, max_list_entries);
         errdefer hard_forks.deinit(allocator);
 
-        const stakes = try Stakes.random(allocator, rand, max_list_entries);
+        const stakes = try Stakes.initRandom(allocator, random, max_list_entries);
         errdefer stakes.deinit(allocator);
 
-        const unused_accounts = try UnusedAccounts.random(rand, allocator, max_list_entries);
+        const unused_accounts = try UnusedAccounts.initRandom(random, allocator, max_list_entries);
         errdefer unused_accounts.deinit(allocator);
 
-        const epoch_stakes = try epochStakeMapRandom(rand, allocator, max_list_entries);
+        const epoch_stakes = try epochStakeMapRandom(random, allocator, max_list_entries);
         errdefer epochStakeMapDeinit(epoch_stakes, allocator);
 
         return .{
             .blockhash_queue = blockhash_queue,
             .ancestors = ancestors,
-            .hash = Hash.random(rand),
-            .parent_hash = Hash.random(rand),
-            .parent_slot = rand.int(Slot),
+            .hash = Hash.initRandom(random),
+            .parent_hash = Hash.initRandom(random),
+            .parent_slot = random.int(Slot),
             .hard_forks = hard_forks,
-            .transaction_count = rand.int(u64),
-            .tick_height = rand.int(u64),
-            .signature_count = rand.int(u64),
-            .capitalization = rand.int(u64),
-            .max_tick_height = rand.int(u64),
-            .hashes_per_tick = if (rand.boolean()) rand.int(u64) else null,
-            .ticks_per_slot = rand.int(u64),
-            .ns_per_slot = rand.int(u128),
-            .genesis_creation_time = rand.int(sig.accounts_db.genesis_config.UnixTimestamp),
-            .slots_per_year = @bitCast(rand.int(u64)),
-            .accounts_data_len = rand.int(u64),
-            .slot = rand.int(Slot),
-            .epoch = rand.int(Epoch),
-            .block_height = rand.int(u64),
-            .collector_id = Pubkey.random(rand),
-            .collector_fees = rand.int(u64),
-            .fee_calculator = FeeCalculator.random(rand),
-            .fee_rate_governor = FeeRateGovernor.random(rand),
-            .collected_rent = rand.int(u64),
-            .rent_collector = RentCollector.random(rand),
-            .epoch_schedule = EpochSchedule.random(rand),
-            .inflation = Inflation.random(rand),
+            .transaction_count = random.int(u64),
+            .tick_height = random.int(u64),
+            .signature_count = random.int(u64),
+            .capitalization = random.int(u64),
+            .max_tick_height = random.int(u64),
+            .hashes_per_tick = if (random.boolean()) random.int(u64) else null,
+            .ticks_per_slot = random.int(u64),
+            .ns_per_slot = random.int(u128),
+            .genesis_creation_time = random.int(sig.accounts_db.genesis_config.UnixTimestamp),
+            .slots_per_year = @bitCast(random.int(u64)),
```

Comment:
If you could call `random.float(f64)` here please.

Reply:
done b4ffcdef27060b8d7bca2cf85e36c714f9db647c